frontend/src/components/porting/port.schema.json

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
Does the file need to be copied here or could there be some common folder the backend and frontend can import from?